app/controllers/devise/registrations/registrations_controller.rb

```diff
@@ -0,0 +1,167 @@
+class Devise::Registrations::RegistrationsController < DeviseController
+  prepend_before_action :require_no_authentication, only: [:new, :create, :cancel]
+  prepend_before_action :authenticate_scope!, only: [:edit, :update, :destroy]
+  prepend_before_action :set_minimum_password_length, only: [:new, :edit]
+
+  # GET /resource/sign_up
```

Comment:
this comment doesn't actually hold true, it's `/users/sign_up`. Can we remove it? it's not really adding any value and we can find this information by running `rails routes`